Library/Formula/atlassian-confluence.rb

```diff
@@ -0,0 +1,64 @@
+# Formula for local Confluence testing
+
+class AtlassianConfluence < Formula
+  homepage "https://www.atlassian.com/software/confluence/"
+  url "http://www.atlassian.com/software/confluence/downloads/binary/atlassian-confluence-5.6.5.tar.gz"
+  sha1 "4999940f0b0837961d72310128d8376c4025c504"
+
+  depends_on :java => "1.7"
+
+  def install
+    # TODO
```

Comment:
Sorry! No, i forgot to remove this line and brew audit did not catch it
either. Tested and works.
Am 06.01.2015 18:46 schrieb "Misty De Meo" <notifications@github.com>:

> In Library/Formula/atlassian-confluence.rb
> <https://github.com/Homebrew/homebrew/pull/35607#discussion-diff-22537627>
> :
>
> > @@ -0,0 +1,64 @@
> > +# Formula for local Confluence testing
> > +
> > +class AtlassianConfluence < Formula
> > +  homepage "https://www.atlassian.com/software/confluence/"
> > +  url "http://www.atlassian.com/software/confluence/downloads/binary/atlassian-confluence-5.6.5.tar.gz"
> > +  sha1 "4999940f0b0837961d72310128d8376c4025c504"
> > +
> > +  depends_on :java => "1.7"
> > +
> > +  def install
> > +    # TODO
>
> Is there still anything TODO here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/35607/files#r22537627>.
>